### PR TITLE
FIX: White screen on boot

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/e380gaa8b5fw29385g9vmcp4fr

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/e8g989gj63uft11xhmy2axkvb0

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ 

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ 

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ 

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ 

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ 

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ 

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ 

(posted automatically)

File: class/app-storage.js [hunk 1]
```diff
@@ -88,6 +88,19 @@ export class AppStorage {
...
+      return data;
+    } catch (e) {
+      console.log('getData error:');
+      console.log(e);
+      const data = { wallets: [], tx_metadata: [] };
+      await this.setItem('data', JSON.stringify(data));
+      return data;
+    }
+  }
+
```
Comment (Overtorment) [inline class/app-storage.js]: so the problem was with biometrics? then revert this. otherwise it can occasionally purge poor unlucky users wallets...

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ 

(posted automatically)

File: class/app-storage.js [hunk 2]
```diff
@@ -455,4 +455,4 @@ export class AppStorage {
   async sleep(ms) {
     return new Promise(resolve => setTimeout(resolve, ms));
   }
-}
+}
```
Comment (Overtorment) [inline class/app-storage.js]: Hmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm  🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔

**Comment by canselcik** [comment]
Just downloaded the update from Play Store and happy to see that this fixes https://github.com/BlueWallet/BlueWallet/issues/708.

**Pull request merged**